src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -44,13 +44,23 @@ export const VetTecContactInformation = ({ institution }) => (
     {/* Production flag for 19534 */}
     {!environment.isProduction() &&
       institution.schoolCertifyingOfficials[0] && (
-        <div className="additional-information row vads-u-margin-y--4">
-          {VetTecScoContact(
-            institution.schoolCertifyingOfficials[0],
-            'School certifying officials',
+        <React.Fragment>
```

Comment:
why use `<React.Fragment>` why not have this be its own div?

Reply:
I can change that to a `div` if preferred, I didn't really see a huge advantage or disadvantage either way here.